src/storage/config_manager.rs

```diff
@@ -78,18 +73,26 @@ impl<EK: Engine, L: LockManager> ConfigManager for StorageConfigManger<EK, L> {
             if let Some(v) = flow_control.remove("enable") {
                 let enable: bool = v.into();
                 if enable {
-                    for cf in self.kvdb.cf_names() {
-                        self.kvdb
-                            .set_options_cf(cf, &[("disable_write_stall", "true")])
-                            .unwrap();
-                    }
+                    self.tablet_factory.for_each_opened_tablet(
```

Comment:
If you can compute the string value ahead of time, then you can only write one loop.

Reply:
> If you can compute the string value ahead of time, then you can only write one loop.
Good point. I followed the original code but it indeed can be simplified.